### PR TITLE
Makes stun baton two words

### PR DESCRIPTION
**Pull request by cacogen**
## About The Pull Request
Makes stun baton two words

## Why It's Good For The Game
I dunno, but this has bugged me for awhile

## Changelog
:cl: cacogen
spellcheck: Makes stun baton two words
/:cl:

### CODE REVIEW

File: code/__DEFINES/inventory.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 //ITEM INVENTORY WEIGHT, FOR w_class
 #define WEIGHT_CLASS_TINY     1 //Usually items smaller then a human hand, ex: Playing Cards, Lighter, Scalpel, Coins/Money
 #define WEIGHT_CLASS_SMALL    2 //Pockets can hold small and tiny items, ex: Flashlight, Multitool, Grenades, GPS Device
-#define WEIGHT_CLASS_NORMAL   3 //Standard backpacks can carry tiny, small & normal items, ex: Fire extinguisher, Stunbaton, Gas Mask, Metal Sheets
+#define WEIGHT_CLASS_NORMAL   3 //Standard backpacks can carry tiny, small & normal items, ex: Fire extinguisher, stun baton, Gas Mask, Metal Sheets
```
Comment (4dplanner) [inline code/__DEFINES/inventory.dm]: The rest of these are capitalised

Comment (subject217) [reply code/__DEFINES/inventory.dm]: But should it be Stun Baton or Stun baton? :thinking:

Comment (4dplanner) [reply code/__DEFINES/inventory.dm]: Why is there both Fire extinguisher and Gas Mask in there, deary me

Comment (81Denton) [reply code/__DEFINES/inventory.dm]: sTuN baTOn

**Comment by subject217** [comment]
broke: capitalizing your one word change to match the rest of the comments in a file
woke: editing every other word so you don't have to capitalize the one you changed

**Comment by cacogen** [comment]
A long time ago item names were capitalised in the game like that before everything that wasn't a proper noun was shifted to lowercase so pronouns worked properly (e.g. [the] flashlight vs. Flashlight). Those comments are probably a holdover from that time.

**Review by ShizCalev — APPROVED** [review]

**Comment by HomocidlePancakes** [comment]
I will still call it stunbaton IC. Fight me faggot.

**Comment by ExcessiveUseOfCobblestone** [comment]
the idea that everything was capitalized hurts

**Comment by cacogen** [comment]
@HomocidlePancakes 
The next phase of the plan involves making stun batons explode if their holder uses the term "stunbaton"

**Pull request merged**